.circleci/config.yml

```diff
@@ -3,9 +3,10 @@ version: 2
 jobs:
   deploy:
     docker:
-      - image: google/cloud-sdk:251.0.0
+      - image: google/cloud-sdk:latest
```

Comment:
At the time I went with `251` it was the latest (`252` has since come out). Do you think this made a difference, or was the `build-essential` install below the key? I think it is in general better practice to use stable tags so we don't run into weird stuff like this in the future.

Reply:
In general I agree with fixing versions, but apt-get/Debian/Ubuntu mirrors get stale and older Docker images will inevitably point to stale mirrors leading to errors using `apt-get install`. This project is so lightweight I would tradeoff potential API changes in favor of CircleCI breaking.

`build-essential` is unrelated although it installs `make` so removing `build-essential` will make the `make` call further down fail.

Reply:
Ah, I wonder if we could also avoid further issues by pinning to a stable tag and running an `apt-get update`? Pointing to latest should be fine as well.

I agree that `make` and other such tools are essential: looks like I accidentally removed `build-essential` when futzing around with ubuntu repository mirrors...